llarp/ev/ev_kqueue.hpp

```diff
@@ -286,17 +286,33 @@ namespace llarp
     setup()
     {
       llarp::LogDebug("set up tunif");
-      if(tuntap_start(tunif, TUNTAP_MODE_TUNNEL, TUNTAP_ID_ANY) == -1)
+
+      if(tuntap_start(tunif, TUNTAP_MODE_TUNNEL, 0) == -1)
```

Comment:
why did this change from `TUNTAP_ID_ANY` to `0`?

Reply:
bad conflict merge? It's nothing I want.

Reply:
no idea how that got there honestly